Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Use non-slim image as includes GCC
+FROM python:3.10-bullseye
+
+WORKDIR /app
+COPY requirements.txt requirements.txt
+# Install git to download utils library
+RUN apt update && apt -yq install git 
+RUN python3 -m pip install --upgrade pip && pip install -r requirements.txt
+COPY . .
+
+EXPOSE 8080
+ENV FLASK_ENV=development
+
+CMD ["flask", "run", "--host", "0.0.0.0", "--port", "8080"]
```

Comment:
So this could use gunicorn as that's available on this repo - although I'm going to update the paths when I implement it using the utils repo (rather than locally) so I can update it then. But perhaps be worth check it all works with it too?

Reply:
My thinking here is that we are going to use the dockerfile primarily for local development. If you want to test with gunicorn you could still override the docker run command.